resources/views/articles/edit.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends('layout.layout')
+
+@section('title')
+    Article
+@endsection
+
+@section('content')
+    <main class="container mt-5 px-5">
+        <section class="container border rounded px-0">
+            <h4 class="border-bottom px-4 py-2 bg-secondary bg-opacity-10">Edit Article</h4>
+            <form action="{{ route('articles.update', ['article' => $article->id]) }}" method="POST" class="container py-3"
```

Comment:
When using route model binding it's just enough to pass `$article` here. Don't need to write everything unless it's really needed.

Reply:
didn't know that that is valid